blockchain/blockindex.go

```diff
@@ -260,6 +260,16 @@ func (node *blockNode) RelativeAncestorCtx(distance int32) HeaderCtx {
 	return ancestor
 }
 
+// IsAncestor returns if the other node is an ancestor of this block node.
+func (node *blockNode) IsAncestor(otherNode *blockNode) bool {
+	ancestor := node.Ancestor(otherNode.height)
```

Comment:
This can return `nil`, so we should do a nil check here.

Reply:
Addressed in the latest push